check-all-vpn-exits.sh

```diff
@@ -14,13 +14,17 @@ fi
 # Name of runfile
 RUN_FILE="/run/$(basename $(readlink -f $0))"
 
+# Check for run file
+if [ -f "$RUN_FILE" ]; then
+  echo 'Runfile does exist!' >&2
+  exit 1
+fi
+
 # Script finish job
 function finish {
-  if [ $? -eq 0 ]; then
```

Comment:
Stimmt. In diesem Kontext nicht. Ich hatte den Block aus einem anderen Script kopiert. Da machte er Sinn :)